site/docs/tutorials/guide-to-v1.mdx

```diff
@@ -0,0 +1,63 @@
+---
+name: Guide to version 1.0
+route: /tutorials/guide-to-v1
+menu: Tutorials
+---
+
+import { Notification } from "hds-react";
+
+import LargeParagraph from "../../src/components/LargeParagraph";
+import Link from "../../src/components/Link";
+
+# HDS guide to start using version 1.0
+
+<LargeParagraph>
+    Here you will find some useful information to make your transition to HDS 1.0 as smooth as possible.
+</LargeParagraph>
+
+<Notification label="TLDR (Too long, didn't read)" className="siteNotification">
+<b>Developers:</b> Update deprecated Dropdown to Select or Combobox component. Do not use deprecated LabelText React property, use Label instead.
+<br /><br />
+<b>Designers:</b> Update your Sketch to version 71.
+</Notification>
+
+## How does this update affect my work?
+The biggest change is that Helsinki Design System is now officially in production and not in beta anymore. In most cases this does not affect your work. It gives you certainty that you can use the components in your own projects. Most of the HDS components have now been marked as Stable. You can review the full list of the current components and their statuses in the [Component overview page](/components).
+
+
+Moving onto production also means that HDS is part of continuous depelopment in the City of Helsinki, and the team is doing their best to provide good quality library components that help building accessible, user friendly and consistent digital services.
```

Comment:
> Moving onto production also means that HDS is part of continuous depelopment in the City of Helsinki, and the team is doing their best to provide good quality library components that help building accessible, user friendly and consistent digital services.

Moving onto production also means that HDS is part of continuous development in the City of Helsinki. The team is doing their best to provide good quality library components that help to build accessible, user-friendly, and consistent digital services.